examples/stress_tests/bevymark.rs

```diff
@@ -60,6 +60,10 @@ struct Args {
     /// the number of different textures from which to randomly select the material color. 0 means no textures.
     #[argh(option, default = "1")]
     material_texture_count: usize,
+
+    /// use a random rather than sequential z value for birds
+    #[argh(switch)]
+    random_z: bool,
```

Comment:
I feel like I want this to default to true.

Reply:
I had thought about instead doing a `z_mode` enum to test the "reverse sorted" case, but I was having a bit of trouble justifying it.

What do you think?

Reply:
I suppose that could trigger a worst case for some sorting algorithms, but I would hope not for any that we would use? Not sure. Random is fine for now imo.